bundle/deploy/terraform/showplanfile.go

```diff
@@ -34,8 +51,9 @@ func GetActions(changes []*tfjson.ResourceChange) []deployplan.Action {
 
 		group, ok := TerraformToGroupName[rc.Type]
 		if !ok {
-			// Happens for databricks_grant, databricks_permissions, databricks_secrets_acl.
-			// These are automatically created by DABs, no need to show them.
+			if !isSilentlyUpdated(rc.Type) {
+				log.Warnf(ctx, "unknown resource type '%s'", rc.Type)
```

Comment:
a few questions: 
 - When does this happen?
 - Can we write an acceptance test?
 - Why not go for hard error?